.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
I would rename the name of the file to what is doing the job (because we could say all workflows are ci-ci jobs)

Something like : `build-publish-main.yml`

Reply:
renamed to `build-publish-next.yml`